var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -216,9 +217,6 @@ def url_for_version(self, version):
         return url.format(version.dotted, version.underscored)
 
     def determine_toolset(self, spec):
-        if spec.satisfies("platform=darwin"):
-            return 'darwin'
```

Comment:
On 1.71.0, this causes the bootstrapping process to crash with the message "no toolset named darwin". Using the default "clang" toolset seems to work fine. Can someone who is more well versed in the history of boost review this and let me know if we still need to add this toolset for older versions?

Reply:
According to `git blame -p boost`, it looks like @tgamblin @scheibelp @chissg @davydden @becker33 are the primary contributors to the `boost` package. Is anyone interested in becoming an official maintainer of this package so I know who to contact in the future?

Reply:
@eschnett it looks like you added this snippet of code in #297. Maybe you can comment?

Reply:
I have very limited time these days for Spack, I used to fix Boost so that it works with deal.II. Sorry, can't maintain it.

Reply:
@svenevs I think you also use Boost on macOS if you want to chime in.